modules/order.js

```diff
@@ -0,0 +1,57 @@
+import {store} from "../Store/store.js";
+import Hamburger from './hamburgers.js'
+import Salads from "./salads.js";
+import Beverages from "./beverages.js";
+import CreateMenu from "./extendClass.js";
+
+class Order extends CreateMenu {
+    constructor() {
+        super();
+        this.isPay = false;
+        this.hamburger = new Hamburger();
+        this.salads = new Salads();
+        this.drinks = new Beverages();
+    }
+
+    addPositions(...args) {
+        if (this.isPay === true)
+            return console.log('cant add');
+        store.push(...args);
+    }
+
+    calculateCalories() {
+        this.caloriesCounter = 0;
+        this.caloriesCounter = this.hamburger.calculateCalories() + this.salads.calculateCalories() + this.drinks.calculateCalories();
```

Comment:
Probably it's better to think about more universal method to calculate calories/price? 
Imagine situation that we will have addition positions like fry potato/fish dish/ice cream... You have a good solution about creation new modules, because it's easy to add new entity, but such functions will be calculated with sum of many entities...